scripts/zkai_attention_kv_two_head_seq32_fused_softmax_table_native_gate.py

```
@@ -0,0 +1,600 @@
+            "proof_backend_version": FUSED_BACKEND_VERSION,
+            "proof_schema_version": FUSED_PROOF_SCHEMA_VERSION,
+            "statement_version": FUSED_STATEMENT_VERSION,
+            "semantic_scope": FUSED_SEMANTIC_SCOPE,
+            "decision": DECISION.replace("TWO_HEAD_SEQ32_", ""),
+            "target_id": FUSED_TARGET_ID,
+            "verifier_domain": FUSED_VERIFIER_DOMAIN,
+        },
+        "fused envelope",
+    )
+    if envelope.get("source_input") != source_input:
+        raise AttentionKvTwoHeadSeq32FusedSoftmaxTableGateError("fused source input split-brain drift")
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)